src/browserlib/extract-algorithms.mjs

```diff
@@ -413,6 +432,10 @@ function getAlgorithmInfo(algo, context) {
     if (container && !context?.nested) {
       if (container.getAttribute('data-algorithm')) {
         info.name = normalize(container.getAttribute('data-algorithm'));
+        if (container.getAttribute('data-algorithm-for')) {
```

Comment:
Is that convention documented anywhere? I didn't know about it

Reply:
Not that I'm aware of. Is `data-algorithm` documented anywhere in practice? Bikeshed and ReSpec talk about the `algorithm` class and attribute, not about the resulting markup.